<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Code Tracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>711</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>711</width>
       <height>671</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="activitiesGroupBox">
        <property name="title">
         <string>Activities</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>691</width>
           <height>194</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTreeView" name="activitiesTable"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="addActivityButton">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="delActivityButton">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <zorder>layoutWidget</zorder>
        <zorder>changesetsGroupBox</zorder>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="changesetsGroupBox">
        <property name="title">
         <string>Changesets</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>691</width>
           <height>194</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTreeView" name="changesetsTable"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="addChangesetButton">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="delChangesetButton">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="testChangesGroupBox">
        <property name="title">
         <string>Test Changes</string>
        </property>
        <widget class="QTreeView" name="testChangesTable">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>601</width>
           <height>192</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="devComboBox">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="devLabel">
     <property name="geometry">
      <rect>
       <x>495</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Developer:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRefresh">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuRefresh"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
